lms/djangoapps/email_marketing/signals.py

```diff
@@ -50,12 +50,12 @@ def add_email_marketing_cookies(sender, response=None, user=None,
         'vars': {'last_login_date': datetime.datetime.now().strftime("%Y-%m-%d")}
     }
 
-    # get sailthru_content cookie to capture usage before logon
+    # get edx_sailthru_content cookie to capture usage before logon
     request = crum.get_current_request()
     if request:
-        sailthru_content = request.COOKIES.get('sailthru_content')
```

Comment:
#nitpick rename the variable to a meaningful one. Like `anonymous_interest_cookie`